sources/packages/web/src/components/generic/TooltipIcon.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <v-tooltip :location="location">
+    <template #activator="{ props }">
+      <v-icon
+        icon="fa:fas fa-circle-question"
+        class="mx-2"
+        color="secondary"
+        size="16"
+        v-bind="props"
+      />
+    </template>
+    <slot></slot>
+  </v-tooltip>
+</template>
+
+<script lang="ts">
+export default {
+  props: {
+    location: {
+      type: String,
+      required: false,
+      default: "right",
```

Comment:
mm, I think agreed not to set aby default location right?

Reply:
We tested it and even by setting default location the responsiveness was not affected and hence the default location property was restored.